src/lib/flatex.ts

```diff
@@ -12,14 +12,15 @@ export type Command = {
 export type Credentials = {
   tokenId: string;
   windowId: string;
+  url: string;
 };
 
 export function getDocumentRows(): Element[] {
   return [...Array.from(document.querySelectorAll('tr[data-wt-click="dokument_download"]'))];
 }
 
-export async function getCredentials(): Promise<Credentials> {
-  const res = await fetch('https://konto.flatex.at/banking-flatex.at/documentArchiveListFormAction.do');
+export async function getCredentials(url: string): Promise<Credentials> {
```

Comment:
I just discovered that Flatex exposes a `webcore` instance on the window object that seems to expose the necessary information. 

```js
const tokenId = window.webcore.getTokenId();
const windowId = webcore.getWindowManagement().getCurrentWindowId()
```

Could you please verify this for DE? 🙏🏻

That might remove the need for passing the `url` as an argument.